source/localizable/templates/links.md

```diff
@@ -101,18 +101,6 @@ In the above example, the model hook for `PhotoRoute` will run with `params.phot
 `CommentRoute` _won't_ run since you supplied a model object for the `comment` segment. The comment's id will
 populate the url according to `CommentRoute`'s `serialize` hook.
 
-### Setting query-params
-
-The `query-params` helper can be used to set query params on a link:
-
-```handlebars
-// Explicitly set target query params
-{{#link-to "posts" (query-params direction="asc")}}Sort{{/link-to}}
-
-// Binding is also supported
-{{#link-to "posts" (query-params direction=otherDirection)}}Sort{{/link-to}}
-```
-
```

Comment:
@locks, looks like this PR killed off this section that got merged recently (and there were some other changes in here too). Was your branch not fully up to date with master?